pycqed/analysis_v2/tomography_execute.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from pycqed.analysis import fitting_models as fit_mods
+from pycqed.analysis import analysis_toolbox as a_tools
+import pycqed.analysis_v2.base_analysis as ba
+# import dataprep for tomography module
+# import tomography module
+from pycqed.analysis_v2 import tomography_dataprep as dataprep
+from pycqed.analysis_v2 import tomography_V2 as tomography_V2
+from pycqed.analysis import measurement_analysis as ma 
+try:
+	import qutip as qt
+except ImportError as e:
+	logging.warning('Could not import qutip, tomo code will not work')
+
+class TomographyExecute(object):
```

Comment:
Wait !?! what does this file do? Isn't the measurement analysis object supposed to do that?